python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_branin_function.py

```diff
@@ -0,0 +1,226 @@
+"""
+Kriging : metamodel of the Branin-Hoo function
+==============================================
+"""
+# %%
+# As a popular use case in optimization we briefly present the construction of a metamodel
+# of the Branin (also refered to as Branin-Hoo) function.
+#
+
+# %%
+import openturns as ot
+import openturns.viewer as viewer
+import openturns.viewer as otv
+from openturns.usecases import branin_function as branin_function
+from matplotlib import pylab as plt
+
+
+
+# %%
+# We load the Branin-Hoo model from the usecases module and use the model (stored in `objectiveFunction`)
+bm = branin_function.BraninModel()
+model = bm.objectiveFunction
+
+# %%
+# We shall represent this 2D function with isolines. We set the number of isolines to a maximum of 10
+# thanks to the following `ResourceMap` key :
+ot.ResourceMap_SetAsUnsignedInteger("Contour-DefaultLevelsNumber",10)
```

Comment:
its best to write ResourceMap.SetAsUnsignedInteger (no _) to show it's a method of ResourceMap